lava/lava-job-definitions/testplans/psa-arch-tests.yaml

```diff
@@ -0,0 +1,28 @@
+{% extends "shared/templates/base.yaml" %}
+
+{% set job_name = "PSA architecture tests" %}
+{% set lxc_creation = true %}
+{% set lxc_name = "psa-arch-tests-lxc" %}
+
+{% block testplan %}
+- test:
+    timeout:
+      minutes: 3
```

Comment:
Is this enough to run the entire job? It has to install mbl-cli and run the psa-archs.
To be on the safe side st it to 10 minutes: sometimes LAVA slaves are busy and they can get a little bit slow when running LXC containers.

Reply:
Thanks for the info. I've increased the timeout to 10min in https://github.com/ARMmbed/mbl-tools/pull/246/commits/6b41a7398c7c59988c144ea64db1c5f5a4322acb